ViewTemplates/Totpmfa/setup.blade.php

```diff
@@ -20,38 +20,35 @@
 $svg = str_replace('<?xml version="1.0" encoding="UTF-8"?>', '', $svg);
 ?>
 
-<div class="row mb-3">
-    <div class="col-sm-9 offset-sm-3">
-        <div class="card">
-            <p class="h3 card-header">
-                Set up your authenticator software
-            </p>
-            <div class="card-body d-flex flex-column flex-lg-row gap-3 gap-lg-0 align-items-start justify-content-center">
-                {{ $svg }}
-                <div>
-                    <p>
-                        Depending on your authenticator software, do one of the following:
-                    </p>
-                    <ul>
-                        <li>
-                            Scan this QR Code&trade;.
-                        </li>
-                        <li>
-                            Use this <a href="{{ $uri }}">link</a>.
-                        </li>
-                        <li>
-                            Enter the Secret <code>{{ $secret }}</code>
-                        </li>
-                    </ul>
-                    <p>
-                        Enter the generated 6-digit code into the field below, and click on “Save”.
-                    </p>
-                    <p class="text-info small">
-                        <span class="fa fa-fw fa-info-circle" aria-hidden="true"></span>
-                        Looking for compatible authenticator software? We have tested with <a href="https://keepassxc.org/">KeePassXC</a>, <a href="https://strongboxsafe.com/">Strongbox</a>, <a href="https://1password.com/">1Password</a>, <a href="https://en.wikipedia.org/wiki/Google_Authenticator">Google Authenticator</a>, <a href="https://authy.com/">Twilio Authy</a>, and the built-in feature in macOS / iOS / iPadOS.
-                    </p>
-                </div>
-            </div>
-        </div>
-    </div>
-</div>
\ No newline at end of file
+<div class="card">
```

Comment:
The indentation here was intentional. Do not remove it.

Reply:
The indention was removed here because it was added higher up the chain in `ViewTemplates/Mfamethods/form.blade.php` to ensure that the other message was also indented.

### before
![image](https://github.com/akeeba/panopticon/assets/1296369/143671a2-4aaf-4853-ad29-9f1b5caa4261)

![image](https://github.com/akeeba/panopticon/assets/1296369/4078a212-07e7-4fc5-a15d-a8ebfaaa5807)

### after
![image](https://github.com/akeeba/panopticon/assets/1296369/86c06d96-6f3a-42cf-9ea2-5ebe3b074d69)

![image](https://github.com/akeeba/panopticon/assets/1296369/652ebaa2-46b7-41f9-8107-1f71fe10ac4e)

Reply:
There was a missing comment of mine which would have made more sense.

The correct approach is to change `src/Library/MultiFactorAuth/Plugin/PassKeys.php`, removing the pre_message from onMfaGetSetup and adding the formatted HTML for the message in ViewTemplates/Passkeysmfa/register.blade.php

Reply:
Addressed with commit [edb3dd1](https://github.com/akeeba/panopticon/pull/469/commits/edb3dd1d345e784cf8addd296dcd8ea940ddccae)